lib/command/configMigrate.js

```diff
@@ -8,8 +8,8 @@ const util = require('util');
 const print = require('../output').print;
 const success = require('../output').success;
 const error = require('../output').error;
-const fileExists = require('../utils').fileExists;
-const getTestRoot = require('./utils').getTestRoot;
+const { fileExists } = require('../utils');
```

Comment:
`const { fileExists, getTestRoot } = require('../utils');`

Reply:
./utils and ../utils they are not the same